pages/create.js

```diff
@@ -0,0 +1,18 @@
+import { useRouter } from "next/router";
+import ActivityForm from "@/components/ActivityForm";
+
+export default function Form({ addActivity }) {
+  const router = useRouter();
+
+  function handleSubmit(data) {
+    addActivity(data);
+    router.push("/");
+  }
+
+  return (
+    <>
+      <h1 id="favourite-activity">Add activity</h1>
```

Comment:
For what purpose do you use the id attribute? Did you want to connect the form with the heading to five the form an accessible name? This would be wonderful, but you need to set the attribute `aria-labelledby` to you form element in your `ActivityForm` do set this connection. You should then also add this for your edit page. :)